app/actions/EventActions.ts

```diff
@@ -375,13 +370,14 @@ export function unfollow(followId: EntityId, eventId: EntityId) {
   });
 }
 
-export function isUserFollowing(eventId: EntityId) {
-  return callAPI<boolean>({
+export function isUserFollowing(eventId: EntityId, currentUserId: EntityId) {
+  return callAPI({
     types: Event.IS_USER_FOLLOWING,
     endpoint: `/followers-event/?target=${eventId}`,
-    schema: [followersEventSchema],
     method: 'GET',
     meta: {
+      eventId,
+      currentUserId,
       errorMessage: 'Henting av interesse feilet',
```

Comment:
Also, this feedback seems misleading

Reply:
Or when I think about it, I suppose following an event is showing "interesse". I was imagining a company interest, and assumed it was koked

Reply:
```suggestion
      errorMessage: 'Henting av stjernemarkering feilet',
```
```suggestion
      errorMessage: 'Henting av følgerstatus feilet',
```